de/django_forms/README.md

```diff
@@ -436,13 +436,13 @@ Mal sehen, ob das alles auch auf PythonAnywhere funktioniert. Zeit für ein weit
 
 {% filename %}PythonAnywhere command-line{% endfilename %}
 
-    $ cd ~/<your-pythonanywhere-username>.pythonanywhere.com
+    $ cd ~/<deine pythonanywere-Domain>.pythonanywhere.com
     $ git pull
```

Comment:
pythonanywere --> pythonanywhere

I would suggest "deine-PythonAnywhere-Domain" or "deine-pythonanywhere-domain", since it is not possible to have spaces in a domain name. I think that's why the English version uses "-" as separators.

Reply:
oops